libs/core/src/lib/multi-input/multi-input-mobile/multi-input-mobile.component.ts

```diff
@@ -32,6 +32,8 @@ export class MultiInputMobileComponent
     extends MobileModeBase<MultiInputInterface>
     implements OnInit, AfterViewInit, OnDestroy
 {
+    allItemSelected: boolean;
```

Comment:
If that's the part of the public API please add the comment to the field, otherwise please prefix name with `_` & add /** @hidden */.